src/app/Dashboard/Dashboard.tsx

```diff
@@ -37,11 +37,21 @@
  */
 import * as React from 'react';
 import { TargetView } from '@app/TargetView/TargetView';
+import { AutomatedAnalysisCard } from './AutomatedAnalysis/AutomatedAnalysisCard';
+import { Grid, GridItem } from '@patternfly/react-core';
 
 export interface DashboardProps {}
 
 export const Dashboard: React.FunctionComponent<DashboardProps> = (props) => {
-  return <TargetView pageTitle="Dashboard" compactSelect={false} hideEmptyState />;
+  return (
+    <TargetView pageTitle="Dashboard" compactSelect={false} hideEmptyState>
+      <Grid hasGutter>
+        <GridItem span={6}>
```

Comment:
Need adjustments for screen-size. We might as well let the automatic analysis to span all columns if screen size hits a certain point. This avoids issue like below (maybe not entirely but mostly so we need to play around with it).

![Screenshot from 2022-11-29 15-52-54](https://user-images.githubusercontent.com/68053619/204645546-f64ac83d-ac4e-4702-b613-132b89ac370f.png)

Reply:
Will need to account for the sidecard space.